website/source/guides/external/lxc.html.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: "guides"
+page_title: "LXC"
+sidebar_current: "guides-external"
+description: |-
+  Guide for using LXC external task driver plugin.
+---
+
+## LXC
+
+The `lxc` driver provides an interface for using LXC for running application containers. You can download the external LXC driver [here][lxc_driver_download]. This guide is compatible with Nomad 0.9 and above. If you are using an older version of Nomad, see the [LXC][lxc-docs] driver documentation.
```

Comment:
the alignment here seems to be off, should be within 80 characters per line to make it easier to read.